test/test_dropbear.py

```diff
@@ -21,11 +21,13 @@ def dropbear(request):
 
 	# split so that "dropbearmulti dropbear" works
 	args = opt.dropbear.split() + [
-		"-p", LOCALADDR, # bind locally only
```

Comment:
Huh, I wonder how it worked previously

Reply:
This resulted in these kind of params:
```
-p 127.0.0.5 -p 2244
```
The first was treated as invalid port (which it is), and disregarded. The second port was bound to all addresses, so it worked.
tjk :)